api/app/tests/fba/test_fba_endpoint.py

```diff
@@ -140,16 +206,35 @@ async def mock_hfi_thresholds(*_, **__):
 
 
 async def mock_sfms_fuel_types(*_, **__):
-    return [(SFMSFuelType(id=1, fuel_type_id=1, fuel_type_code="C2", description="test fuel type c2"),)]
+    return [
+        SFMSFuelType(id=1, fuel_type_id=1, fuel_type_code="C2", description="test fuel type c2")
```

Comment:
Real change here: Now returning an array of `SFMSFuelType` objects instead of an array of tuples of `SFMSFuelType` objects.